apps/coscrad-frontend/src/components/resources/bibliographic-references/book-bibliographic-reference-detail.full-view.presenter.tsx

```diff
@@ -0,0 +1,52 @@
+import {
+    IBibliographicReferenceViewModel,
+    IBookBibliographicReferenceData,
+    IValueAndDisplay,
+} from '@coscrad/api-interfaces';
+import { Card, CardContent, CardHeader, Divider } from '@mui/material';
+import { SinglePropertyPresenter } from '../../../utils/generic-components';
+
+export const BookBibliographicReferenceDetailFullViewPresenter = ({
+    id,
+    data,
+}: IBibliographicReferenceViewModel<IBookBibliographicReferenceData>): JSX.Element => {
+    const { title, abstract, year, publisher, place, url, numberOfPages, isbn } = data;
+
+    const labelsAndValues: IValueAndDisplay<unknown>[] = (
+        [
```

Comment:
This logic is repeated between all spatial feature detail presenters. It might be nice to break this out. At very least add a todo.